README.md

```diff
@@ -117,15 +118,6 @@ Badges look really great, but they're not always easy to set up. Take a look at
 ## About the code
```

Comment:
Lines 109-113: Why is it only Jenkins that has the trademark symbol?

Reply:
How should it look? What are the standards? I'm not familiar with the doc standards on this so I'd need some more help from you here. All have it? None? Do we have to look it up?

Reply:
It's desirable to include the trademark symbol for a product at list once on a page. Can you add the symbols based on what you see here?
https://www.mathworks.com/help/matlab/matlab_prog/continuous-integration-with-matlab-on-ci-platforms.html

Azure® DevOps, CircleCI®, Jenkins®,  GitHub® Actions, GitLab® CI/CD